cirq-core/cirq/vis/heatmap.py

```diff
@@ -387,16 +426,18 @@ def plot(
         original_config = copy.deepcopy(self._config)
         self.update_config(**kwargs)
         qubits = set([q for qubits in self._value_map.keys() for q in qubits])
+        collection_options: Dict[str, Any] = {"cmap": "binary"}
+        highlighted_qubits = kwargs.get("highlighted_qubits")
+        if highlighted_qubits is None:
```

Comment:
the code from `main` has
```py3
            collection_options={
                'cmap': 'binary',
                'linewidths': 2,
                'edgecolor': 'lightgrey',
                'linestyle': 'dashed',
            },
```

but this logic means that collection_options will only become ` collection_options: Dict[str, Any] = {"cmap": "binary"}` if the `highlited_qubits is None`

Reply:
Isn't it the opposite? If `highlighted_qubits` is None, then all the key-values from collection_options are retained. Only when `highlited_qubits` is not None does collection_options only contain 'cmap': 'binary'

Reply:
yes you are right .. for some reason I kept thinking `is not`